### ISSUE TITLE
Error on fresh build

### ISSUE THREAD

**Issue by brycereynolds** [jbosse/ignite-expo-boilerplate#16]
I'm running a fresh install of ignite + this boilerplate and I'm getting the following when it tries to build the js bundle...

```

Unable to resolve ../../../../App" from ".//node_modules/react-native-scripts/build/bin/crna-entry.js`: could not resolve `/Users/bryce/MyLatestCreation/App' as a file nor as a folder","name":"UnableToResolveError","type":"UnableToResolveError","errors":[{}]},"type":"bundling_error"}"

```

**ignite** `2.0.0`
**node** `8.6.0`

Thoughts? Something I missed maybe?

**Comment by jbosse** [comment]
I did a `ignite new Bryce -b ignite-expo`...
```
-----------------------------------------------
  (                  )   (                   
  )\ )   (        ( /(   )\ )    *   )       
 (()/(   )\ )     )\()) (()/(  ` )  /(   (   
  /(_)) (()/(    ((_)\   /(_))  ( )(_))  )\  
 (_))    /(_))_   _((_) (_))   (_(_())  ((_) 
 |_ _|  (_)) __| | \| | |_ _|  |_   _|  | __|
  | |     | (_ | | .` |  | |     | |    | _| 
 |___|     \___| |_|\_| |___|    |_|    |___|
-----------------------------------------------

An unfair headstart for your React Native apps.
https://infinite.red/ignite

-----------------------------------------------

🔥 igniting app Bryce
✔ using the Infinite Red boilerplate v2 (code name 'Andross') forked with Expo.io
✔ added React Native 0.50.4 in 30.96s
? Would you like Ignite Development Screens? Yes
? What internationalization library will you use? react-native-i18n
? What animation library will you use? react-native-animatable
? Would you like to include redux-persist? Yes
✔ added ignite-expo in 41.03s
✔ added ignite-dev-screens-expo in 29.15s
✔ added ignite-i18n in 23.99s
✔ added ignite-animatable in 17.42s
✔ added ignite-redux-persist in 35.77s
✔ added ignite-standard in 44.07s
✔ configured git
✔ ignited Bryce in 246.04s

    Ignite CLI ignited Bryce in 246.04s

    To get started:

      cd Bryce
      yarn start

    Read the walkthrough at https://github.com/infinitered/ignite-ir-boilerplate/blob/master/readme.md#boilerplate-walkthrough

    Learn about working with Expo.io at https://docs.expo.io/versions/latest/index.html

    Need additional help? Join our Slack community at http://community.infinite.red.

    Now get cooking! 🍽
  
Jimmys-MacBook-Pro:play-n-learn jbosse$ 
```

then I did `cd Bryce` and `yarn start` and iOS launched.

**Comment by jbosse** [comment]
@brycereynolds were you able to get this working?

**Comment by Panthro** [comment]
Had the same issue with node 9+, downgraded to node8 and it worked fine.

Using windows btw

**Comment by Kaisaurus** [comment]
On a fresh install getting the following error: 
```
17:42:54: TypeError: undefined is not a function (evaluating 'addListener')

This error is located at:
    in CardStack (at CardStackTransitioner.js:67)
    in RCTView (at View.js:71)
    in View (at Transitioner.js:142)
    in Transitioner (at CardStackTransitioner.js:19)
    in CardStackTransitioner (at StackNavigator.js:39)
    in Unknown (at createNavigator.js:13)
    in Navigator (at createNavigationContainer.js:227)
    in NavigationContainer (at ReduxNavigation.js:14)
    in ReduxNavigation (created by Connect(ReduxNavigation))
    in Connect(ReduxNavigation) (at RootContainer.js:23)
    in RCTView (at View.js:71)
    in View (at RootContainer.js:21)
    in RootContainer (created by Connect(RootContainer))
    in Connect(RootContainer) (at App.js:24)
    in Provider (at App.js:23)
    in App
    in RCTView (at View.js:71)
    in View
    in Unknown (at registerRootComponent.js:35)
    in RootErrorBoundary (at registerRootComponent.js:34)
    in ExpoRootComponent (at renderApplication.js:35)
    in RCTView (at View.js:71)
    in View (at AppContainer.js:102)
    in RCTView (at View.js:71)
    in View (at AppContainer.js:122)
    in AppContainer (at renderApplication.js:34)
- node_modules/react-navigation/src/getChildEventSubscriber.js:95:16 in <unknown>
- node_modules/react-navigation/src/getChildEventSubscriber.js:94:40 in getChildEventSubscriber
- node_modules/react-navigation/src/views/CardStack/CardStack.js:445:6 in _getScreenDetails
* null:null in render
- node_modules/react-proxy/modules/createPrototypeProxy.js:44:35 in proxiedMethod
- node_modules/react-native/Libraries/Renderer/ReactNativeRenderer-dev.js:7866:21 in finishClassComponent
- ... 24 more stack frames from framework internals
```
node 8.9.4
ignite 2.0.0

Any ideas on what's going wrong?

**Comment by LeungMingFai** [comment]
Problem with react-navigation and new redux version

Following changes fix this problem. 😄 
https://github.com/infinitered/ignite-ir-boilerplate/pull/185/files

**Comment by BobbyBakes** [comment]
@Kaisaurus I'm having the same issue. Any resolution?

**Comment by jbosse** [comment]
There is a pr but it needs some trivial changes to pass ci but I think it’s already out of date and I haven’t had a chance to merge the fork to make the changes. 😕

**Comment by jbosse** [comment]
@LeungMingFai, @BobbyBakes I have merged and pushed the PR to 0.25.4. I also pushed 0.26.0-alpha with a bump to expo 26 and react 16.3 alpha.
